src/NuGet.Core/NuGet.Common/NuGetEnvironment.cs

```diff
@@ -85,22 +76,23 @@ public static string GetFolderPath(SpecialFolder folder)
                                 () => GetEnvironmentVariable("PROGRAMDATA"),
                                 () => GetEnvironmentVariable("ALLUSERSPROFILE"));
                         }
+                        else if (RuntimeEnvironmentHelper.IsMacOSX)
+                        {
+                            return Path.Combine(@"/","Library", "Application Support");
+                        }
                         else
                         {
-                            return "/usr/share";
+                            return Path.Combine(@"/","etc","opt");
```

Comment:
Same here, `/etc/opt`, though I don't know that many products actually use the `/etc/opt` root in reality.